cloud-native-runtimes/install-cnrt.hbs.md

```diff
@@ -147,8 +143,7 @@ To install Cloud Native Runtimes:
    Without these credentials, attempts to start a service fail with a timeout and the pods report that they are unable to pull the `queue-proxy` image.
```

Comment:
I'm not able to comment on line 138, but it has another dead link (`/set-up-namespaces.md`)